src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -145,6 +147,23 @@ function useSidebarMetadataFetcher(
         [api, onApiError, file, deleteMetadataInstanceSuccessCallback],
     );
 
+    const updateMetadataInstance = () => {
+        // to be implemented in the next ticket
+    };
```

Comment:
I just noticed this, we shouldnt add in code that is not part of the ticket unless its absolutely necessary.  In other [PR](https://github.com/box/box-ui-elements/pull/3672), this function is being handled with a different name

Reply:
Ok thanks, I deleted this empty function